src/test/java/com/google/api/codegen/protoannotations/testdata/ruby_library.baseline

```diff
@@ -6228,6 +6308,11 @@ end
 
 # DO NOT EDIT! This is a generated sample ("Request",  "test_write_to_file")
 
+# sample-metadata
+#   title: test writing fields to files
+#   description: Testing write fields to files.
+#   bundle exec ruby samples/v1/publish_series_request_test_write_to_file.rb
```

Comment:
`ruby some_file.rb` should be fine, but I think most of the time people invoke ruby code through `bundle exec`